src/api/cart.js

```diff
@@ -60,26 +75,18 @@ export default class extends Base {
      * @param {Function} callback
      */
     getCartQuantity(options = {}, callback) {
-        this.getCart(options, (err, response) => {
+        this.getCartSummary(options, (err, response) => {
             if (err) {
                 return callback(err);
             }
             let quantity = 0;
-            if (response && response.status !== 404) {
-                const cart = response;
-                const lineItemQuantities = [
-                    cart.lineItems.physicalItems,
-                    cart.lineItems.digitalItems,
-                    cart.lineItems.customItems,
-                ].reduce((a, b) => a.concat(b))
-                    .filter((lineItem) => !lineItem.parentId)
-                    .map((lineItem) => lineItem.quantity)
-                    .reduce((accumulator, lineItemQuantity) => accumulator + lineItemQuantity, 0);
-                const giftCertificateQuantity = cart.lineItems.giftCertificates.length;
-                quantity = lineItemQuantities + giftCertificateQuantity;
+            if (response
+                && response.status !== 204
+                && response.total_quantity) {
+                quantity = response.total_quantity;
```

Comment:
@junedkazi as the response for this endpoint is changing, I hope this conditional will handle these 3 cases:

- []
- {}
- HTTP 204 (final state of the API)